packages/swing-store/src/snapStore.js

```diff
@@ -203,7 +215,17 @@ export function makeSnapStore(
               compressedSize,
               compressedSnapshot,
             );
-
+            noteExport(
+              artifactName({ vatID, endPos }),
+              JSON.stringify({
+                vatID,
+                endPos,
+                hash: h,
+                uncompressedSize,
+                compressedSize,
```

Comment:
Let's keep the property names the same wherever they are copied to, so nothing should say just `size` if there's uncertainty about whether we're talking compressed or uncompressed.

Definitely agree that compressed size should not be exported. I can agree that uncompressed size doesn't strictly need to be included, as the hash nails down the contents, but if something in the transport layer is truncating artifacts (probably to zero bytes), or appending/prepending some junk, or e.g. base64 encoding them and not reversing the transformation at the other end, then it'd be nice to compare lengths before complaining that the hashes don't match. So I'd be inclined to include `uncompressedSize` in the exported data, next to the hash.

The importer should recompute the non-consensus `compressedSize` in their own DB, for the analytics purposes I mentioned earlier.

Reply:
While I can see the debugging utility of `uncompressedSized` for heap snapshots, I'd like to be explicit about what a corresponding value for transcript span would be. Currently it is the sum of UCS-2 chars in transcript entries, including the line terminations. This is a very JS centric value, and doesn't correspond neither to the number of Unicode characters in the entries, nor to the number of bytes in the utf-8 encoded artifact. The best for exports would be to use the number of utf-8 bytes, but that should only be computed during exports to avoid unnecessary conversion during save.

Reply:
This was just a coding mistake -- when we settled on the principle that only the uncompressed form of artifacts should be visible externally I missed this bit when converting things over.